packages/blocks/src/__internal__/utils/query.ts

```diff
@@ -231,18 +229,27 @@ export function getModelsByRange(range: Range): BaseBlockModel[] {
       commonAncestor = parentElement;
     }
   }
+
   const intersectedModels: BaseBlockModel[] = [];
-  const blockElementArray = commonAncestor.querySelectorAll(`[${ATTR}]`);
-  if (blockElementArray.length > 1) {
-    blockElementArray.forEach(ele => {
-      const block = ele as ContainerBlock;
-      assertExists(block.model);
-      const blockElement = getBlockElementByModel(block.model);
```

Comment:
Tried to removed this. Not sure why it queries the block element DOM by the DOM back again.